Documentation/release-notes/6000.md

```diff
@@ -0,0 +1,11 @@
+#### Application and library build and deployment
+
+- [GitHub Issue 5432](https://github.com/xamarin/xamarin-android/issues/5432):
+  In a .NET 6 application, native libraries can now be contained in directories
+  which match Android *Resource Identifiers*, not just the legacy Android ABI
```

Comment:
Should this be:

```suggestion
  which match Android *Runtime Identifiers*, not just the legacy Android ABI
```

Could also link to: https://github.com/dotnet/designs/blob/main/accepted/2021/net6.0-tfms/net6.0-tfms.md#rids